server/management/appointments/appointments.php

```diff
@@ -112,12 +102,18 @@ function rescheduleAppointment($appointmentId, $appointmentTimeId) {
 		// Reset fields in the associated serviced appointment if applicable
 		$query = "UPDATE ServicedAppointment
 			SET timeIn = NULL, timeReturnedPapers = NULL, timeAppointmentStarted = NULL, timeAppointmentEnded = NULL,
-				completed = FALSE, notCompletedDescription = NULL, servicedByStation = NULL
+				completed = FALSE, servicedByStation = NULL
 			WHERE appointmentId = ?";
 		$stmt = $DB_CONN->prepare($query);
 
 		$success = $stmt->execute(array($appointmentId));
 		if ($success == false) throw new Exception();
+
+		// Add a note to the appointment saying it was rescheduled
+		$noteAccessor = new NoteAccessor();
```

Comment:
@CollinsSpencer Also thoughts on this automatic note addition? Yes or no?

Reply:
I think that is a really good idea!